Material.Blazor/Material.Blazor.csproj

```diff
@@ -64,6 +64,9 @@
   </ItemGroup>
 
   <ItemGroup>
+    <None Update="Scripts\BatchingJsRuntime.ts">
```

Comment:
Not sure what this actually does. The compilation did the expected thing on my machine (TM)